package/etc/conf.d/conflib/syslog/app-syslog-a10.conf

```diff
@@ -0,0 +1,20 @@
+block parser app-syslog-a10() {
+    channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index('netops')
+                source('a10networks:vThunder')
+                sourcetype('a10networks:vThunder:syslog')
+                vendor('a10networks')
+                product('vThunder')
+            );
+        };
+    };
+};
+
+application app-syslog-a10[sc4s-syslog-pgm] {
+    filter {
+        program('a10logd' type(string) flags(prefix));
```

Comment:
@rjha-splunk @mstopa-splunk Is it okay or `program('a10logd' type(string) flags(prefix)) and message('[audit log]'  type(string)  flags(prefix))` would be better? I just really worry about that `a10logd` might be suite for different devices for that vendor

Reply:
`a10logd` will be sufficient